coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4226,27 +4288,38 @@ def gelu(context, node):
     context.add(res)
 
 
-@register_torch_op(torch_alias=["slice"])
-def _slice(context, node):
-    inputs = _get_inputs(context, node, expected=5)
+@register_torch_op(torch_alias=["_slice", "slice_copy.tensor"])
+def slice(context, node):
+    inputs = _get_inputs(
+        context,
+        node,
+        expected={TorchFrontend.TORCHSCRIPT : 5},
+        min_expected={TorchFrontend.EDGEIR : 1},
+    )
     x = inputs[0]
-    dim = inputs[1].val
+    dim = 0 if len(inputs) < 2 else inputs[1].val
 
-    if inputs[2] and inputs[2].val is not None:
-        start = inputs[2].val
-    elif isinstance(inputs[2], Var):
-        start = inputs[2]
-    else:
-        start = 0
-
-    if inputs[3] and inputs[3].val is not None:
-        end = inputs[3].val
-    elif isinstance(inputs[3], Var):
-        end = inputs[3]
-    else:
-        end = None
+    start = 0
+    if len(inputs) > 2 and inputs[2] is not None:
```

Comment:
Just curious,
I think in MIL, slice_by_index already support negative indices,
if we comment those two handling out,
what error were you getting?

Reply:
TLDR: Actually, the error was fixed, no need to treat negative indices now 👍 will remove these negative indices treatment and kick off another CI

## Context
I used to encounter an error from out-of-range index:
* PyTorch use `INT64_MAX` as default `end`, in which case the last element is included in the slice result
* We use `int32` during conversion, so `INT64_MAX` becomes `-1`
* This `-1` confuses `mb.slice_by_index`, excluding the last element from the slice result
* This is not an issue anymore, since I revised `mb.const` for `int64`, to clip out-of-range value to `int32` max (or min)